src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\FrameworkBundle\Controller\ControllerResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\DefaultValueResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\RequestAttributeValueResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\RequestValueResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\ServiceValueResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\SessionValueResolver;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\VariadicValueResolver;
+use Symfony\Component\HttpKernel\Controller\ErrorController;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadataFactory;
+use Symfony\Component\HttpKernel\EventListener\DisallowRobotsIndexingListener;
+use Symfony\Component\HttpKernel\EventListener\ErrorListener;
+use Symfony\Component\HttpKernel\EventListener\LocaleListener;
+use Symfony\Component\HttpKernel\EventListener\ResponseListener;
+use Symfony\Component\HttpKernel\EventListener\StreamedResponseListener;
+use Symfony\Component\HttpKernel\EventListener\ValidateRequestListener;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->defaults()->private()
+
+        ->set('controller_resolver', ControllerResolver::class)
+        ->args([
+            service('service_container'),
+            service('logger')->ignoreOnInvalid(),
+        ])
+        ->tag('monolog.logger', ['channel' => 'request'])
+
+        ->set('argument_metadata_factory', ArgumentMetadataFactory::class)
+
+        ->set('argument_resolver', ArgumentResolver::class)
+        ->args([
+            service('argument_metadata_factory'),
```

Comment:
let's add `abstract_arg()` for empty argument definition in XML.

In this case, can be something like this `abstract_arg('argument value resolvers')`

Ref: https://github.com/symfony/symfony/pull/36778/files#diff-c9620897a07dd9bcada6305d3623c687R151